### ISSUE TITLE
Conditional Buy and sell

### ISSUE THREAD

**Issue by shargon** [Domysee/BittrexSharp#22]
Should be nice have this features

PD: Great job

**Comment by Domysee** [comment]
I agree it would be nice, unfortunately I don't have time right now to maintain BittrexSharp. 
If you could implement it, I'd be happy to accept a pull request!

**Comment by shargon** [comment]
It is on Api v2.0 but there is no documentation about this... could you provide me this documentation?

**Comment by Domysee** [comment]
Sorry I also just have the Bittrex documentation. If it is not in there, then I'm afraid I can't help you.
